### ISSUE TITLE
NanoPi Fire 3 LTS compatibility

### ISSUE THREAD

**Issue by 222464** [friendlyarm/nexell_linux_platform#10]
Hello,

I am trying to use the CSI camera on the NanoPi Fire 3 LTS. With some modifications, I am able to compile this library up until linking to the pre-built libnxvidrc.so. I need 64 bit, but that prebuilt shared library is 32 bit. I have looked for this file (google and nexell website), but without success.

Any suggestions for what I can do? Thanks!

**Comment by slaminger** [comment]
Yes. Don't ask lol. Nobody but a handful of us own these and we are on our own. I've got mednafen and retro arch running tho.

**Comment by johlim** [comment]
> Hello,
> 
> I am trying to use the CSI camera on the NanoPi Fire 3 LTS. With some modifications, I am able to compile this library up until linking to the pre-built libnxvidrc.so. I need 64 bit, but that prebuilt shared library is 32 bit. I have looked for this file (google and nexell website), but without success.
> 
> Any suggestions for what I can do? Thanks!

nexell_linux_platform is 32bit arm arch. , so 64bit is not supported

**Comment by crouse** [comment]
编译成功找不到 /dev/ion 请问咋办？

**Comment by friendlyarm** [comment]
> Hello,
> 
> I am trying to use the CSI camera on the NanoPi Fire 3 LTS. With some modifications, I am able to compile this library up until linking to the pre-built libnxvidrc.so. I need 64 bit, but that prebuilt shared library is 32 bit. I have looked for this file (google and nexell website), but without success.
> 
> Any suggestions for what I can do? Thanks!

It is 32-bit armhf demo app and can only be used with linux-3.4.y
For linux-4.4.y, you should use gstreamer

**Comment by friendlyarm** [comment]
> 编译成功找不到 /dev/ion 请问咋办？

你的内核是 4.4 还是 3.4 ？
这个仓库只能用于 3.4 内核，4.4内核没有 /dev/ion ，也不支持在4.4内核环境下运行。

**Comment by yeyun11** [comment]
> 
请问一下4.4及以上该如何使用nanocams？预装直接执行没发现nanocams
